src/pybind/mgr/dashboard/frontend/src/app/shared/classes/tab-helper.class.ts

```diff
@@ -0,0 +1,28 @@
+import { OnInit, ViewChild } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+
+import { TabDirective, TabsetComponent } from 'ngx-bootstrap/tabs';
+
+export class TabHelper implements OnInit {
+  @ViewChild(TabsetComponent, { static: true })
+  tabset: TabsetComponent;
+
+  constructor(protected route: ActivatedRoute, protected router: Router) {}
+
+  ngOnInit() {
+    const fragment = this.route.snapshot.fragment;
```

Comment:
I assume that fragment should actually be passed to the class (?)

Reply:
`fragment`is a member variable of `ActivatedRouteSnapshot`, but it must be set in a `this.router.navigate(['/monitoring'], { fragment: 'silences' });` command.